apps/filecoin-site/src/app/(homepage)/page.tsx

```diff
@@ -167,7 +178,32 @@ export default function Home() {
           description="Insights, updates, ecosystem spotlights, and community stories, directly from the teams building Filecoin."
           cta={<Button href={PATHS.BLOG.path}>View all articles</Button>}
         >
-          TODO
+          <CardGrid as="ul" cols="lgThree">
+            {featuredBlogPosts.map((post: BlogPost) => {
+              const { title, slug, excerpt, categories, image, author, date } =
+                post
+              return (
+                <BlogCard
+                  key={title}
+                  title={title}
+                  description={excerpt}
+                  author={author}
+                  date={date}
+                  tags={categories}
+                  image={
+                    image && {
+                      src: image.url,
+                      alt: title,
+                    }
+                  }
+                  cta={{
+                    href: `${PATHS.BLOG.path}/${slug}`,
+                    ariaLabel: `Read more about ${title}`,
```

Comment:
Both these properties seem to be the same always? Wonder if we can just pass the `slug`..

Reply:
Yea they are pretty much the same - so we could get rid of the `title` property, and create `slugToTitle` function?